### PR TITLE
fix: Add padding to diff code blocks

### PR DESCRIPTION
**Pull request by chargome**
fixes [#10510](https://github.com/getsentry/sentry-docs/issues/10510)

### CODE REVIEW

File: src/components/codeBlock/code-blocks.module.scss [hunk 1]
```diff
@@ -73,12 +76,16 @@
   :global(.diff-inserted),
   :global(.inserted) {
     background-color: rgba(16, 185, 129, 0.2);
+    margin-left: -12px;
+    width: calc(100% + 12px);
   }
 
   /* Set deleted line (-) color */
```
Comment (szokeasaurusrex) [inline src/components/codeBlock/code-blocks.module.scss:83]: Maybe update this comment (and the comment for the above changes) to reflect that we are also setting the padding here

**Review by szokeasaurusrex — COMMENTED** [review]
I will take a look at the preview once it is ready. Since I am not super experienced with CSS, it would be good if someone with more experience could take a look at the code changes

**Comment by chargome** [comment]
Alignment works the same way, but the padding is now added automatically. So if a diff block currently contains manual indentations, the padding will be added on top. There are ~100 diff blocks in the project, I can check if they need to be updated.

**Review by lforst — APPROVED** [review]

**Comment by ezio-melotti** [comment]
The selector is a bit too generic and adds padding after other operators in the code examples.  I'm trying to fix that in:
* #11052

**Pull request merged**